### ISSUE TITLE
Update navbar to be solid on scroll

### ISSUE THREAD

**Issue by stephchin** [railsbridgesd/railsbridgesd#3]

**Comment by mokolabs** [comment]
FYI... I'm changing our issue names to be imperative present tense so we can use them as future commit messages when we complete the issue.

**Comment by stephchin** [comment]
Ok I actually made the navbar fixed already. I also made it transparent, but when you scroll passed the background image, it needs to become solid so that the text isn't overlapping.

**Comment by sslampa** [comment]
I just pushed a new branch, navbar, to github. I'm not sure if that's what you were thinking of, but I wanted to at least start contributing to the project.

**Comment by stephchin** [comment]
Close! Clearly I'm not good at articulating my issues haha... I wanted transparent over the photo and then solid over the content. I just switched what you did so that it turns solid after it hits the bottom of intro. Thanks!!

**Comment by mokolabs** [comment]
I made a few tweaks to this:

- While it's clever to grab the height of the intro image and use that as the basis for triggering a navar background color change, it's not that precise. By using a hard coded number for this, we can be more exact.

- Also, ideally, this behavior should not rely on any global variables to function, we can remove the `imageHeight` var that's outside the scope of our `$(document).on("scroll"...` function.

- When we change the background, let's do that by adding and removing a class to the navbar. That way, we can tweak how the background looks in CSS without manually setting lots of properties in our javascript code.

- I added a bottom border to the navbar when we're scrolling... so that we can always distinguish between the navbar and the content (even though both have a white background).

- I switched our CSS selector to just `.navbar` for now.
